File: src/main/java/rk/cinema/repository/HallRepository.java
package rk.cinema.repository;

import io.micronaut.data.jdbc.annotation.JdbcRepository;
import io.micronaut.data.model.query.builder.sql.Dialect;
import io.micronaut.data.repository.CrudRepository;
import rk.cinema.model.Hall;

@JdbcRepository(dialect = Dialect.POSTGRES)
public interface HallRepository extends CrudRepository<Hall, Long> {
}

File: src/main/kotlin/rk/powermilk/cinema/repository/HallRepository.kt
package rk.powermilk.cinema.repository

import io.micronaut.data.jdbc.annotation.JdbcRepository
import io.micronaut.data.model.query.builder.sql.Dialect
import io.micronaut.data.repository.CrudRepository
import rk.powermilk.cinema.model.Hall

@JdbcRepository(dialect = Dialect.POSTGRES)
interface HallRepository : CrudRepository<Hall, Long>
